### DISCUSSION TITLE
use @pixi/events，if move the sprite quickly will outside the sprite

### DISCUSSION THREAD

**Discussion by freezyh** [pixijs/pixijs#8687] [Q&A]
``` 
import { Sprite } from 'pixi.js'
 class Rabbit extends Sprite { 
    constructor(app) {
        this.dragging = false
        this.stone = new Sprite(texture)
        this.stone.position.set(200, 200);
        this.stone.anchor.set(0.5);
        this.stone.height = 60;
        this.stone.width = 60;
        this.stone.zIndex = 99;
        this.stone.interactive = true;
        this.stone.buttonMode = true;
        this.app.stage.addChild(this.stone);
    }
    initEvent() {
        this.stone.on('pointerdown', this.pointerDown.bind(this));
        this.stone.on('pointerup', this.pointerUp.bind(this));
        this.stone.on('pointermove', this.pointerMove.bind(this));
    }
    pointerDown(e) {
        this.dragging = true;
        this.pointerMove(e);
    }

    pointerMove(e) {
        if (this.dragging) {
            this.stone.position.copyFrom(e.data.global)
            this.app.renderer.render(this.stone);
        }
    }
    pointerUp(e) {
        this.dragging = false;
    }
}
```

### Current Behavior
 move the sprite quickly will outside the sprite，can't move the sprite again， you should tap the sprite
you can view the code and download, run in mobile browser, move the sprite quickly
https://gitee.com/nicefree/eat-food/blob/master/src/components/Rabbit.js
### Environment

- **`pixi.js` version**: 6.5.4
- **`@pixi/events` version**: 6.5.4

**Comment by bigtimebuddy** [comment]
This isn't as much a bug as your approach for implementing pointermove. 

It's better to add the listener on pointerdown and remove it on pointerup. Also, add the pointermove to the whole view/stage/window, not to the object itself. If your mouse is moving faster than the render, it will not get the pointermove event. It's more stable to add it to the view or a nested container instead.
